docs/management/optimization/memory-optimization.md

```diff
@@ -11,15 +11,15 @@ aliases: [
 
 ## Special encoding of small aggregate data types
 
-Since Redis 2.2 many data types are optimized to use less space up to a certain size.
+Since Valkey 2.2 many data types are optimized to use less space up to a certain size.
```

Comment:
From this point to 7.2 I think we should remove it. There is no Valkey 2.2-7.0 (and _Redis_ 2.2 was from like.. a decade ago?